packages/shift-next/src/pages/slug.js

```diff
@@ -22,9 +22,10 @@ const slugRequest = (slug) => {
       slugs: 'resource_type,resource_id,active,slug'
     }
   }
-  const query = qs.stringify(queryObject)
+
   return {
-    endpoint: `/getSlug/?${query}`
+    endpoint: '/getSlug',
+    query: queryObject
```

Comment:
Before we were assigning `query` for no reason, and never using it on line 36 as there was no such thing as `request.query`